docs/requirements-insiders.txt

```diff
@@ -1,7 +1,8 @@
 PyYAML==6.0.2
 ruff==0.11.9
 mkdocs==1.6.1
-mkdocs-material @ git+ssh://git@github.com/astral-sh/mkdocs-material-insiders.git@39da7a5e761410349e9a1b8abf593b0cdd5453ff
+mkdocs-material @ https://github.com/astral-sh/mkdocs-material-insiders.git#39da7a5e761410349e9a1b8abf593b0cdd5453ff
```

Comment:
Does this work? I'm asking because the repository is private under the Astral org on GitHub and cloning that would require the SSH keys when `git+ssh` is being used and username/password for the `https` protocol.

Reply:
I reverted it. I always need to change it because I never set up SSH auth :)